app/views/content/salaries-and-benefits.md

```diff
@@ -87,7 +87,7 @@ You also get other insurance benefits too. You can find out more on [the teacher
 
 ## Support for early career teachers
 
-All teachers are given extra support during their first 2 years in teaching. This helps early career teachers (ECTs) develop their knowledge, teaching skills and working habits.
+All teachers are given extra support during their first 2 years in teaching called [ECF-based training](/support-for-early-career-teachers). This helps early career teachers (ECTs) develop their knowledge, teaching skills and working habits.
 
 This support includes:
```

Comment:
'replaced' rather than 'replaces' NQT?

Reply:
Also - I wonder whether we should remove the link to the GOV.UK ECF page and replace it with one to our ECF page?

Reply:
Agree with 'replaced' rather than 'replaces'.

I think that if we're only linking 'early career framework', we should continue to link to the .gov page because that gives more detail about ECF, whereas our page is more about ECF-based training/ECT induction.

Reply:
Fine with me!